plugins/parquet/include/parquet/operator.hpp

```diff
@@ -198,9 +265,9 @@ class parquet_printer final : public plugin_printer {
         }
       }
       parquet_writer_props_builder.version(
-        ::parquet::ParquetVersion::PARQUET_2_6);
+        ::parquet::ParquetVersion::PARQUET_2_LATEST);
```

Comment:
`2_LATEST` is a recent addition; right now it's the same value, and generally the recommendation is to always write the latest format.